src/dmd/frontend.d

```diff
@@ -99,10 +100,12 @@ Initializes the global variables of the DMD compiler.
 This needs to be done $(I before) calling any function.
 
 Params:
+    handler = a delegate of type DiagnosticHandler to configure what to do with diagnostics (other than printing to console or stderr).
```

Comment:
Minor detail: it's unnecessary to mention the type (`DiagnosticHandler`)  since that's available in the signature.